packages/context-editor/src/style.css

```diff
@@ -148,3 +148,22 @@
 	/* Otherwise the widget button is too hard to click without triggering math */
 	margin-left: 4px;
 }
+
+/* Code block */
+.ProseMirror .codeblock-select-wrapper {
+	position: absolute;
+	left: 0;
+	top: 100%;
+	z-index: 100;
+	opacity: 0;
+	transition: all 0.3s ease;
+}
+.ProseMirror .codeblock-root {
+	position: relative;
+}
+.ProseMirror .codeblock-root:hover .codeblock-select-wrapper {
+	opacity: 1;
+}
+.ProseMirror .codeblock-wrapper {
+	margin-bottom: 16px;
+}
```

Comment:
maybe we could indent the codeblock a lil to make it more inline with the rest of the elements?

```suggestion
.ProseMirror .codeblock-root {
	position: relative;
	margin: 0;
	padding: 0;
	border-width: 0;
}
.ProseMirror .codeblock-root:hover .codeblock-select-wrapper {
	opacity: 1;
}
.ProseMirror .codeblock-wrapper {
	padding-left: 5px;
	padding-top: 5px;
	margin-bottom: 16px;
}
```

before:
<img width="653" alt="image" src="https://github.com/user-attachments/assets/97f354b3-aa96-47f9-9b61-cf6a298bf3e6" />


after:
<img width="619" alt="image" src="https://github.com/user-attachments/assets/03eafe32-64e4-4ee1-b4cc-66ff06ac50b1" />